lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml

```diff
@@ -9,7 +9,16 @@ set_placeholder_input_value:
       xpath: (//input[@placeholder='<placeholder_text>'])[<index>]
     op: send_keys <input_value>
     type: input
-  
+check_alert_warning_for_multiple_receivers:
```

Comment:
It looks much clear if we can rename it to `check_message_when_multiple_receivers_not_configured` or something like this. `not_configured` is key message

Reply:
ok